src/common/timelineEvent.ts

```diff
@@ -61,6 +62,7 @@ export interface CommitEvent {
 }
 
 export interface MergedEvent {
+	id: number;
```

Comment:
There's an existing issue which is preventing id from being passed through - in `parseGraphQLTimelineEvents` in `github/utils.ts`, there are casts on each object pushed to the return array. We should get rid of these casts, because they're breaking type checking there. `id` should be added to those objects